docs/_posts/2020-01-08-533-release.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blogpost
+title:  "5.3.3 Release Announcement"
+section_id: blog
+date:   2020-01-08 00:00:00
+author: Open GEE Development Team
+---
+
+<br />
+
+Hello Open GEE Community!
+ 
+We are happy to announce the official release of Open GEE 5.3.3!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+
+**Enhancements:**
+
+**- Ability to set default database from admin page.** There is now an additional option for 3D databases that allows the user to specify one paritcular database as the "default". This default database iwll be served to the client if the server URL is specified without a database name.
+
+**- View information via getop while gesystemmanager is busy.**  Previously, the utility "getop" could not display information while the system manager was performing major calculations. This restriction has now been removed for most calculations. 
+
+**- Improved memory usage reporting.**  The memory usage estimations for asset and asset version caches have been improved and are now much more accurate. 
+
+**- Miscellaneous security updates.** Supplemental security updates have been made to protect against a variety of potential risks.
+
+**- Documentation improvements.**  Existing documentation has been refined and clarified. Documentation has been added for use of LDAP and for building on fips-enabled machines. 
+
+**- Portable and WMS clients.** Improved ease of use of Portable for WMS clients.
+
+**Upgraded components:**
+
+**- Use of Python 2.7 with EL6-based operating systems.**  EL6 OSs now use Python 2.7 in a virtual environment for increased security. EL7 and Ubuntu are unaffected.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.3.3-XXXX.XX). The full release notes can also be found [here](http://www.opengee.org/geedocs/5.3.3/answer/releaseNotes/relNotesGEE5_3_3.html).
+ 
+Our thanks go out to all of the contributors who helped make this release possible! The next release, Open GEE 5.3.4, is already in progress!
+ 
+Would you like to be part of the project? Please join us on [Slack](http://slack.opengee.org/) and visit [the project’s Github repository](https://github.com/google/earthenterprise). We would love to see you there!
```

Comment:
That Slack URL doesn't work. A user reported that it was bad in the forums a couple weeks ago. @tst-lsavoie was going to send him a direct invitation - he may know the correct one. I have no idea what the correct link is, but https://opengee.slack.com/ looks like a possibility?